vllm/entrypoints/openai/protocol.py

```diff
@@ -532,6 +537,8 @@ class CompletionResponseChoice(OpenAIBaseModel):
             "to stop, None if the completion finished for some other reason "
             "including encountering the EOS token"),
     )
+    prompt_logprobs: Optional[List[Optional[Dict[int, Logprob]]]] = Field(
+        default=None)
```

Comment:
```suggestion
    prompt_logprobs: Optional[List[Optional[Dict[int, Logprob]]]] = None
```

Using `Field` is unnecessary here.